src/components/Themes/ui/PreviewTab/TasksPreview/TasksPreview.tsx

```diff
@@ -0,0 +1,862 @@
+                                                    );
+                                                })}
+                                            </Flex>
+                                        </Flex>
+
+                                        <Flex
+                                            height="200px"
+                                            direction="column"
+                                            position="sticky"
+                                            style={{
+                                                backgroundColor: 'var(--g-color-base-background)',
+                                                bottom: 0,
+                                            }}
+                                        >
+                                            <Divider style={{width: '100%'}} />
+
+                                            <Card
+                                                view="outlined"
+                                                spacing={{mx: 5, mt: 5, px: 2, py: 2}}
+                                                style={{flexGrow: 1}}
+                                            >
+                                                <MarkdownEditorView stickyToolbar editor={editor} />
+                                            </Card>
+
+                                            <Flex
+                                                spacing={{m: 5}}
+                                                alignItems="center"
+                                                justifyContent="space-between"
+                                            >
+                                                <Flex gap={1} alignItems="center">
+                                                    <Text color="secondary">Collaborators</Text>
+                                                    <Button view="flat">
+                                                        <Icon data={Plus} />
+                                                    </Button>
+                                                </Flex>
+
+                                                <Button view="outlined-action" disabled>
+                                                    Send
+                                                </Button>
+                                            </Flex>
+                                        </Flex>
+                                    </Box>
+                                </Card>
+                            </Flex>
+
+                            {/* Footer */}
+                            <Flex
+                                spacing={{px: 5, py: 2}}
+                                alignItems="center"
+                                justifyContent="space-between"
+                            >
+                                <Text color="secondary">
+                                    © 2011—${new Date().getFullYear()} Copyright
```

Comment:
`$` is not necessary
<img width="190" alt="image" src="https://github.com/user-attachments/assets/4c6cc5f5-fda9-483f-8c39-e6a46114b7ec" />